### PR TITLE
{bio} [goolf/1.4.10] MView 1.57 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review/test?

File: easybuild/easyconfigs/m/MView/MView-1.49-goolf-1.4.10-Perl-5.16.3.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+dependencies = [('Perl', '5.16.3')]
+
+sanity_check_paths = {
+    'files': ["bin/mview"],
+    'dirs': ["lib"],
+}
+
+modextravars = {
+    'PERL5LIB': '$root/lib'
+    }
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MView/MView-1.49-goolf-1.4.10-Perl-5.16.3.eb]: fix indenting

File: easybuild/easyconfigs/m/MView/MView-1.49-goolf-1.4.10-Perl-5.16.3.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+homepage = 'http://sourceforge.net/projects/bio-mview/'
+description = """ MView reformats the results of a sequence database search or a 
+ multiple alignment, optionally adding HTML markup."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [('http://sourceforge.net/projects/bio-mview/files/bio-mview/mview-%(version)s/', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Perl', '5.16.3')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MView/MView-1.49-goolf-1.4.10-Perl-5.16.3.eb]: maybe define the perl version once and use it both here and in the `versionsuffix`?

File: easybuild/easyconfigs/m/MView/MView-1.57-goolf-1.4.10-Perl-5.16.3.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    'files': ["bin/mview"],
+    'dirs': ["lib"],
+}
+
+modextravars = {
+    'PERL5LIB': '$root/lib'
+    }
+
+moduleclass = 'bio'
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MView/MView-1.57-goolf-1.4.10-Perl-5.16.3.eb]: excess newline

File: easybuild/easyconfigs/m/MView/MView-1.57-goolf-1.4.10-Perl-5.16.3.eb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+dependencies = [('Perl', '5.16.3')]
+
+sanity_check_paths = {
+    'files': ["bin/mview"],
+    'dirs': ["lib"],
+}
+
+modextravars = {
+    'PERL5LIB': '$root/lib'
+    }
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MView/MView-1.57-goolf-1.4.10-Perl-5.16.3.eb]: fix indenting

**Comment by wpoely86** [comment]
looks fine

**Comment by wpoely86** [comment]
@boegel I don't have goolf

**Comment by boegel** [comment]
@wpoely86: maybe give it a shot with --try-toolchain (but let's wait until @pescobar has tackled the remarks)

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 2 out of 2
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/ff07c605cd469d026799 for a full test report.

**Comment by fgeorgatos** [comment]
@pescobar: this has minor style remarks. fix?

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/f4e810e124c381c08aef095e4e5763bc for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/6ed20d9b692635f45d9755cc10ed4464 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/0131b65bd85f3db58fdb1247a68d8cdb for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2160.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/331bcee389ef2e260d5c0052b3565b64 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**